airflow/utils/logging.py

```diff
@@ -71,7 +71,7 @@ def read(self, remote_log_location, return_error=False):
             try:
                 s3_key = self.hook.get_key(remote_log_location)
                 if s3_key:
-                    return s3_key.get_contents_as_string().decode()
+                    return s3_key.get_contents_as_string()
```

Comment:
The problem was spotted with non-ascii logs.

Python 2.7.
chr(215).decode()
UnicodeDecodeError: 'utf8' codec can't decode byte 0xd7 in position 0: unexpected end of data